comps/llms/src/text-generation/opea_llm_microservice.py

```diff
@@ -36,6 +37,7 @@
     host="0.0.0.0",
     port=9000,
 )
+@opea_telemetry
 @register_statistics(names=["opea_service@llm"])
 async def llm_generate(input: Union[LLMParamsDoc, ChatCompletionRequest, SearchedDoc]):
```

Comment:
When `stream=True`. This is case 2) so the duration is not correct. Only correct when `stream=False`.